obs-studio-server/source/nodeobs_service.cpp

```diff
@@ -2173,10 +2173,14 @@ void OBS_service::JSCallbackOutputSignal(void* data, calldata_t* params)
 
 		if (signal.getOutputType().compare("streaming") == 0)
 			output = streamingOutput;
-		else
+		else if (signal.getOutputType().compare("recording") == 0)
 			output = recordingOutput;
+		else if (signal.getOutputType().compare("replay-buffer") == 0)
+			output = replayBufferOutput;
+		else
+			output = nullptr;
 
-		const char* error = obs_output_get_last_error(output);
+		const char* error = (output ? obs_output_get_last_error(output) : "invalid output type");
 		if (error) {
```

Comment:
Seems like this check is not necessary anymore.

Reply:
I added just in case we decide to add another output type without adding an if, but I will remove it to handle the cases that we have now.

Reply:
Ready!